### PR TITLE
Adding additional context for phdi-chart versioning

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
Adds additional context for how the code ensures we are pulling the latest version of phdi-charts.

## Related Issue
Fixes #1932

## Additional Information
There's an additional PR on phdi-charts to give context there, as well.

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Review by bamader — APPROVED** [review]
LGTM! Like the simplicity and duplication between the two files.

File: terraform/aws/implementation/description.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# PHDI Charts
+
+Part of the helm charts used in order to deploy are based on services created and maintained by DIBBs in the [phdi repository](https://github.com/CDCgov/phdi). A helm chart for these respective services is subsequently created in the [phdi-charts repository](https://github.com/CDCgov/phdi-charts) These helm charts ensure that Terraform can build each of these services to be able to deployed online for any individual to access without needing to install and run a given service locally.
```
Comment (m-goggins) [inline terraform/aws/implementation/description.md:3]:
```suggestion
Deployment of DIBBs services in the [phdi repository](https://github.com/CDCgov/phdi) relies on helm charts. A helm chart for each DIBBs service is created in the [phdi-charts repository](https://github.com/CDCgov/phdi-charts) These helm charts ensure that Terraform can build each of these services to be able to deployed online for any individual to access without needing to install and run a given service locally.
```

Comment (m-goggins) [reply terraform/aws/implementation/description.md:3]: I may not have gotten the language exactly right but I was confused by the first sentence.

**Review by m-goggins — APPROVED** [review]
I added a small suggestion to clarify the first sentence but feel free to change if I didn't articulate it correctly since I haven't been as close to the helm charts in a minute.

**Pull request merged**